crates/re_viewer/src/ui/annotations.rs

```diff
@@ -66,7 +70,7 @@ pub enum DefaultColor<'a> {
 }
 
 #[derive(Clone)]
-pub struct ResolvedAnnotationInfo(pub Option<AnnotationInfo>);
+pub struct ResolvedAnnotationInfo(pub Option<ClassId>, pub Option<AnnotationInfo>);
```

Comment:
I find `.0` and `.1` really ugly

```suggestion
pub struct ResolvedAnnotationInfo {
    pub class_id: Option<ClassId>,
    pub info: Option<AnnotationInfo>,
}
```